### DISCUSSION TITLE
Redirection when accessing google cache of gatsby website

### DISCUSSION THREAD

**Discussion by jibin2706** [gatsbyjs/gatsby#35034] [Help]
When accessing the cache version of the website I'm getting redirected
For Eg: `cache:https://www.peppercontent.io` if you visit, you can see the cache version of the site for some time but it redirects to the main page after some seconds.

Upon debugging I found if I disable the webpack-runtime chunk it works as expected. How to fix this?

Also, a related issue is when I access the text-only version of the site, the image component occupies the entire screen (it is because CSS is striped). 
Eg: `https://webcache.googleusercontent.com/search?q=cache:https://www.peppercontent.io/&strip=2&vwsrc=0`

**Comment by Frostbourn** [comment]
Did you found any workaround? I have exact same problem.

**Reply by jibin2706** [reply]
We migrated to next.js
